.travis.yml

```diff
@@ -51,16 +52,18 @@ matrix:
       os: osx
       osx_image: xcode7.3
       compiler: clang
+    - env: CHOST=x86_64-w64-mingw32
+      os: linux
+      compiler: ": Win64 build only"
```

Comment:
it is common practice of running cross builds on travis-ci.
you just need to define "compiler" to something.

and, since travis uses $compiler --version, it is safe to define fake compilers in such way